hack/update-openapi-spec.sh

```diff
@@ -53,6 +53,7 @@ ETCD_HOST=${ETCD_HOST:-127.0.0.1}
 ETCD_PORT=${ETCD_PORT:-2379}
 API_PORT=${API_PORT:-8050}
 API_HOST=${API_HOST:-127.0.0.1}
+API_LOGFILE=/tmp/openapi-api-server.log
```

Comment:
Do we not want to do the same `${FOO:-default}` pattern the rest use?

Reply:
Configuring host/port for etcd and apiserver let's one run against an existing server. I don't think we care for the logfile.

/hole cancel

Reply:
I ended-up updating :-) Thanks!